Makefile

```diff
@@ -0,0 +1,50 @@
+NAME    := munge
+SRC_EXT := rpm
+TARBALL_VERSION = 0.5.13
+VERSION := $(TARBALL_VERSION)-lp151.3.3
+URL_BASE := https://download.opensuse.org/source/distribution/leap/15.1
+URL_BASE := $(URL_BASE)/repo/oss/src
+SOURCE   = $(URL_BASE)/$(NAME)-$(VERSION).src.$(SRC_EXT)
+
```

Comment:
Any reason you are not using the common `Makefile_packaging.mk`?

Reply:
Special case.  Grabbing a LEAP-15.1 package for rebuild SLES-12.3/Open suse 42.3.

Not much of the code ended up the same and I did not want to add more conditionals to Makefile_packaging.mk.

Reply:
The `Makefile_packaging.mk` should be generic package building code for any RPM -- that's the whole point of it.  It shouldn't matter the source or target of the RPM -- all of that code should work regardless.

What kind of conditionals where you having to add to `Makefile_packaging.mk`?

Reply:
Now using Makefile_packaging.mk.  Added the conditionals to it so it can also be used on the older packaging.

Reply:
Converted to use the SUSE "build" program.